test/riemannBackend.js

```diff
@@ -0,0 +1,35 @@
+var test = require("unit.js");
+var config = { riemann: { host: {} } };
+var riemann = new require('../lib/riemann');
+
+describe("riemann-backend", function(){	    
+
+	var riemannBackend = new riemann({}, config, { on: function(){ } });
```

Comment:
If you don't mind just fixing the tabs here (spaces instead of tabs and two spaces per tab that would be great.